### ISSUE TITLE
Reviving sphinxcontrib-ansi

### ISSUE THREAD

**Issue by ssbarnea** [sphinx-contrib/ansi#6]
That is a tracking ticket for actions needed for bringing back the project to being actively maintained:

- [x] @ssbarnea pypi project ownership 
- [ ] @ssbarnea added as core on github. I still need admin as current permissions do not allow me to control webhooks or configuration access to branches. Related to
- [ ] review and eventually merge current PRs
- [ ] configure github actions pipelines
- [ ] bring CI to green on current versions of sphinx
- [ ] team up with others, collective maintenance is better

**Comment by jenisys** [comment]
@ssbarnea 
Are there any other contributors that want to work on this project (except me) ?

**Comment by ssbarnea** [comment]
@jenisys Nope. I send you an invite https://github.com/sphinx-contrib/ansi/invitations

**Comment by jenisys** [comment]
@ssbarnea 
Could you resent the invitation as maintainer?
It was probably expired when I noticed it last time.

In addition, some things needs fixing, like:

* namespace module: must be solved differently by now
* Some ANSI escapes are currently not expected at leads to exceptions (like mentioned in #9)

**Comment by ssbarnea** [comment]
@jenisys You are already a project memeber.
